audio-worklet/src/bitcrusher-demo.js

```diff
@@ -85,16 +113,50 @@ class BitcrusherDemo {
           default: this.masterGain_.gain.value,
           name: 'Volume'
         });
+
+    // GUI controls allow switching between audio worklet and script processor.
+    // The audio worklet node is used by default.
+    this.useAudioWorkletCheckBox_ = document.createElement('input');
+    this.useAudioWorkletCheckBox_.type = 'checkbox';
+    this.useAudioWorkletCheckBox_.checked = true;
```

Comment:
No, you can't make this the default because the AudioWorklet may not be available.  You have to check if it's available first.

I would also have the worklet checkbox disabled if worklets aren't available.